honeybee_vtk/cli/export.py

```diff
@@ -107,11 +112,56 @@ def export():
     '--text-bold', '-tb', is_flag=True, default=False, show_default=True,
     help='Set the text to be bold.'
 )
+@click.option(
+    '--time-step-file-name', '-tfn', type=str, default='', show_default=True,
+    help='Name of the time step file.'
+)
+@click.option(
+    '--periods-file', '-pf', help='File Path to the Periods json file which can be used'
+    ' to define time periods and colors. A period is composed of two Ladybug DateTime'
+    ' objects. First is the start DateTime and second is the End DateTime. Think of'
+    ' these periods as filters of time steps. Images will be generated for these periods'
+    ' only. You can also define colors for corresponding periods.',
+    type=click.Path(exists=True), default=None, show_default=True
+)
+@click.option(
+    '--flat/--gradient', default=True, show_default=True, help='Indicate'
+    ' whether to use flat transparency or gradient transparency in generating time step'
+    ' images.'
+)
+@click.option(
+    '--transparent-images', '-ti', is_flag=True, default=False, show_default=True,
+    help='Indicate whether to generate transparent images or not.'
+)
```

Comment:
To be able to use the flags with Pollination we need them to have two options.

Reply:
Noted.  Turned it into a flag with two options.